barretenberg/ts/jest.config.cjs

```diff
@@ -0,0 +1,12 @@
+/** @type {import('jest').Config} */
```

Comment:
Lets keep the jest config we have in package.json, its common across the monorepo